src/components/KPagination/KPagination.vue

```diff
@@ -1,6 +1,8 @@
 <template>
   <nav
+    ref="kPaginationElement"
     class="k-pagination"
+    :class="{ 'full': !disablePageJump && !offset }"
```

Comment:
nit: since this literally means "full width" can we update the class name to `full-width` so it's obvious?

Reply:
Since "full" doesn't mean anything in the context then, whatever class you need essentially means "stack it on mobile" right? 

What about `page-jump`?

Reply:
I like that, renamed to `page-jump`